tools/cockpit.spec

```diff
@@ -675,7 +675,11 @@ BuildArch: noarch
 Summary: Cockpit user interface for virtual machines
 Requires: cockpit-bridge >= %{required_base}
 Requires: cockpit-system >= %{required_base}
+%if 0%{?rhel} == 7
 Requires: libvirt
+%else
+Requires: libvirt-daemon-kvm or libvirt
```

Comment:
This needs to be put into parentheses, otherwise it looks like three independent dependencies to RPM.